src/Pagination.jsx

```diff
@@ -272,11 +280,27 @@ export default class Pagination extends React.Component {
     let jumpNext = null;
     let firstPager = null;
     let lastPager = null;
+    let gotoButton = null;
 
+    const goButton = props.goButton || (props.showQuickJumper && props.showQuickJumper.goButton);
```

Comment:
Why support `goButton` and `showQuickJumper.goButton` both?

I think `showQuickJumper.goButton` is enough. And it will be nice if `goButton` can used to customize the node.

```jsx
<Pagination showQuickJumper={{ goButton: <Button>xxx</Button> }} />
```